docs/csharp/tutorials/console-webapiclient.md

```diff
@@ -28,7 +28,10 @@ service on GitHub. You'll read information in JSON format, and convert
 that JSON packet into C# objects. Finally, you'll see how to work with
 C# objects.
 
-There are a lot of features in this tutorial. Let’s build them one by one. 
+There are a lot of features in this tutorial. Let’s build them one by one.
+
+If you prefer to follow along with the final sample, you can [download the sample app](https://github.com/dotnet/docs/tree/master/samples/csharp/getting-started/console-webapiclient) for this topic. For download instructions, see [Samples and Tutorials](../../samples-and-tutorials/index.md#viewing-and-downloading-samples).
+
```

Comment:
redundancy in "final sample" and "sample app". Perhaps "with the final sample for this topic, you can download it."